nftest.yml

```diff
@@ -10,29 +10,15 @@ cases:
     nf_script: ./main.nf
     nf_config: test/HaplotypeCaller-37.config
     params_file: test/yamls/HaplotypeCaller-37.yaml
-    skip: false
+    skip: true
     verbose: true
     asserts:
       - script: test/compare-VCFs.sh
-        expect: /hot/project/method/AlgorithmEvaluation/BNCH-000142-GRCh37v38/test/output/TCGA-SARC_10TN-WGS_GRCh37-to-GRCh38/TCGA-SARC_10TN-WGS_GRCh37_gSNP_HaplotypeCaller_StableLift-GRCh38.vcf.gz
-        actual: 'pipeline-StableLift-*/TCGA-SARC_10TN-WGS_GRCh37/StableLift-*/output/TCGA-SARC_10TN-WGS_GRCh37-HaplotypeCaller-stability.vcf.gz'
+        expect: /hot/project/method/AlgorithmEvaluation/BNCH-000142-GRCh37v38/test/expect/TCGA-SARC_10TN-WGS_GRCh37-to-GRCh38/TCGA-SARC_10TN-WGS_GRCh37_gSNP_HaplotypeCaller_StableLift-GRCh38.vcf.gz
```

Comment:
**praise:** I cannot tell you how many times I confused myself with `expect` and `actual` and `output` - at no point did I ever consider renaming this folder. Well done!